tools/progress.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: © 2023 ZeldaRET
+# SPDX-License-Identifier: MIT
```

Comment:
ZeldaRET? :thinking:

Reply:
Actually wasn't sure if I should change that. Probably just Decompals than.